src/app/info/DefaultWikiItemView.tsx

```diff
@@ -0,0 +1,44 @@
+"use client";
+
+import { DbWikiRow } from "@/databaseUtils";
+import { WikiItemAccordionSurface, WikiUpdateDataButton } from "@/app/info/StyleComponents";
+import UpdateIcon from "@mui/icons-material/Update";
+import { Accordion, AccordionDetails, AccordionSummary } from "@mui/material";
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+import { convertContentToElements } from "@/app/info/WikiItems";
+import AdminManagerDependent from "@/app/info/AdminManagerDependent";
+
+interface DefaultViewProps {
+    rowRef: DbWikiRow;
+    setIsInEditModeRef: (isInEditMode: boolean) => void;
```

Comment:
Given that we only ever call this with `true` in this file, can we change it to an argument-less function then pass it through in the calling component?
So here it's
```
interface DefaultViewProps {
    rowRef: DbWikiRow;
    openEditMode: () => void;
...
                <WikiUpdateDataButton
                    onClick={openEditMode}
                >
```
and where this is called:
`<DefaultWikiItemView rowData={row} openEditMode={() => setIsinEditMode(true)} />`